composites/SnippetPreview/components/Button.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+import colors from "../../../style-guide/colors.json";
+
+/**
+ * Returns a basic styled button.
+ *
+ * @param {object} props Component props.
+ *
+ * @returns {ReactElement} Styled button.
+ */
+export const Button = styled.button`
+	display: inline-block;
+	padding: 8px 10px;
+	border: 1px solid ${ colors.$color_button_border };
+	border-radius: 4px;
+	background: ${ colors.$color_button };
+	color: ${ colors.$color_button_text };
+	cursor: pointer;
+	box-sizing: border-box;
+	font-size: inherit;
+	font-family: inherit;
+	font-weight: inherit;
+
+	::-moz-focus-inner {
+		border-width: 0;
+		padding: 0;
+	}
+
+	:focus {
```

Comment:
We should create an `addFocusStyles` function which is used for this button and everywhere where we need this focus style.